bindings/python/CMakeLists.txt

```diff
@@ -215,14 +215,21 @@ install(
   DESTINATION ${Python3_MODULE_PATH}
   PATTERN ".gitignore" EXCLUDE
   PATTERN "*~" EXCLUDE
-  )
-
+)
 install(
   DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/python-mapping-plugins
   DESTINATION share/dlite
   PATTERN ".gitignore" EXCLUDE
   PATTERN "*~" EXCLUDE
-  )
+)
+install(
+  CODE "
+   execute_process(
+     COMMAND ${Python3_EXECUTABLE} setup.py install
+       --prefix=${CMAKE_INSTALL_PREFIX} --force
+     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+   )"
+)
```

Comment:
I do not understand what this has to do with the issue.  It looks to me like this is installing dlite-python. Did we not do that before?

Reply:
> I do not understand what this has to do with the issue. It looks to me like this is installing dlite-python. Did we not do that before?

This install command runs the new setup.py in the bindings/python/ directory. 

The rationale is that we use cmake for building and installing dlite. 
But setuptools knows better how to install python scripts, like how to add a shebang (#!/path/to/python/interpreter/for/current/virtualenv) on the first line. 
To achieve this, cmake calls `setup.py install` to install the scripts.

Reply:
Thank you for the explanation.